packages/react/src/components/TextArea/TextArea.stories.js

```diff
@@ -206,10 +211,23 @@ export const withAILabel = (args) => {
   );
 };
 
-withAILabel.argTypes = {
-  ...sharedArgTypes,
-};
+withAILabel.args = { ...sharedArgs, helperText: 'Optional helper text' };
```

Comment:
```suggestion
withAILabel.args = { helperText: 'Optional helper text' };
```

This goes along with the comment made above about `sharedArgs`. Putting `args` inside of the `export default` block will apply the args to every story, so there will be no need for this variable.